cmd/keeper/keeper.go

```diff
@@ -103,7 +107,21 @@ var defaultPGParameters = pg.Parameters{
 	"hot_standby":             "on",
 }
 
-func (p *PostgresKeeper) getSUConnParams(keeperState *cluster.KeeperState, setPassword bool) pg.ConnParams {
+func (p *PostgresKeeper) getInitialSUPassword() string {
+	var pw string
+
+	if p.cfg.initialSUPasswordFile != "" {
+		pwBytes, err := ioutil.ReadFile(p.cfg.initialSUPasswordFile)
+		if err != nil {
+			log.Errorf("Unable to read password from file %s, error: %s", p.cfg.initialSUPasswordFile, err.Error())
```

Comment:
`log.Errorf("Unable to read password from file %s, error: %v", p.cfg.initialSUPasswordFile, err)` to make it consistent with other errors logging.